README.md

```diff
@@ -45,7 +45,8 @@ Certain versions of Quick and Nimble only support certain versions of Swift. Dep
 
 |Swift version        |Quick version   |Nimble version |
 |:--------------------|:---------------|:--------------|
-|Swift 3              |v1.0.0 or later |v5.0.0 or later|
+|Swift 4.2            |v1.3.2 or later |v7.1.3 or later|
```

Comment:
Should this be `v7.3.1` instead of `v7.1.3`?

Reply:
I'll update this to [v7.3.2](https://github.com/Quick/Nimble/releases/tag/v7.3.2).

Reply:
Updated in a98cf49.